src/client/js/components/Page/Page.jsx

```diff
@@ -50,7 +51,13 @@ function pageFactory(type) {
         {!window.navigator.onLine && <OfflinePage {...props} error={state.error} />}
       </>
     ) : (
-      <main role="main" id="content" tabIndex="-1" aria-hidden={navigationVisible}>
+      <main
+        role="main"
+        id="content"
+        tabIndex="-1"
+        aria-hidden={navigationVisible}
+        className={style({ outline: 'none' })}
```

Comment:
I think we're pushing this css-in-js a little too far :) I mean, when we end up writing the same thing as you would write in the style prop with no obvious advantages... not a critique about this code though, no need to change it